testsuite/tests/service_mesh/auth/conftest.py

```diff
@@ -11,3 +11,9 @@ def no_auth_status_code(staging_gateway):
     if isinstance(staging_gateway, WASMGateway):
         return 403
     return 401
+
+
+@pytest.fixture(scope="module")
+def extension(staging_gateway, service) -> WASMExtension:
```

Comment:
I would move it to the lowest possible conftest/test in this case, `test_authorities`

Reply:
I removed the function as is no longer needed.